Library/Formula/sdl_ttf.rb

```diff
@@ -18,8 +18,13 @@ class SdlTtf < Formula
   depends_on "sdl"
   depends_on "freetype"
 
+  # Fix broken TTF_RenderGlyph_Shaded()
+  # https://bugzilla.libsdl.org/show_bug.cgi?id=1433
+  patch :DATA
+
   def install
     ENV.universal_binary if build.universal?
+    inreplace "SDL_ttf.pc.in", "@prefix@", HOMEBREW_PREFIX
```

Comment:
I'm also in favor of cleaner formula anyways. Okay, now the diff is hosted from [my gist](https://gist.github.com/tomyun/a8d2193b6e18218217c4).